apps/src/code-studio/pd/foorm/FoormLibraryEditorManager.jsx

```diff
@@ -146,20 +208,44 @@ class FoormEditorManager extends React.Component {
           to get started.
         </p>
         <div>
-          <DropdownButton id="load_config" title="Load Form..." className="btn">
+          <DropdownButton
+            id="load_config"
+            title="Load Library..."
+            className="btn"
+          >
             {this.getFormattedConfigurationDropdownOptions()}
           </DropdownButton>
-          <Button onClick={this.initializeEmptyCodeMirror} className="btn">
-            New Form
+          <Button onClick={this.initializeNewLibrary} className="btn">
+            New Library
+          </Button>
+        </div>
+        <div>
+          <DropdownButton
+            id="load_config"
+            title="Load Library Question..."
+            className="btn"
+            disabled={!this.libraryQuestionOptionsAvailable()}
+          >
+            {this.libraryQuestionOptionsAvailable() &&
```

Comment:
Do you need this check here if it's already in `disabled`?

Reply:
Just checked, I think unnecessary. I think I added when I wasn't initializing `availableLibraryQuestionsForCurrentLibrary` in the Redux store as an empty array and was getting errors when trying to map over null/undefined.